### ISSUE TITLE
Motion tracking

### ISSUE THREAD

**Issue by bipsendk** [dbuezas/icsee-ptz#39]
I have a camera with motion tracking, which can be enabled/disabled via the app on my android phone...

Would it be possible to add a sensor/control telling if tracking was enabled or disabled ?

Let me know if I can assist with network traces or similar.

**Comment by dbuezas** [comment]
In the integration click configure for the device of interest and enable experimental entities. You'll get switches for motion tracing

**Comment by martinmain** [comment]
@dbuezas Hi David great app, I have been trying to get the experimental motion switches to work using VMS to monitor and it seems as though the switches are getting toggled but not applied and hence motion detection remains either on or off.

**Comment by dbuezas** [comment]
yeah, the cameras don't quite respect the protocol, each one seems to do their own thing. The only suggestion I have is to reverse engineer which options the official app toggles for that specific camera. If you do code and are willing to put some time into it, look up the asyncio_dvrip.py file

**Comment by martinmain** [comment]
@dbuezas  I am able to do some coding and managed to get the following from the camera using a utility, but not sure this helps?

<img width="748" height="323" alt="Image" src="https://github.com/user-attachments/assets/5b6dbc6e-c2fb-4ad1-966c-72169af1f5e7" />
